app/src/main/java/com/androidstudy/mpesa/Config.kt

```diff
@@ -1,8 +1,22 @@
+/*
+ * Copyright 2021 Juma Allan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.androidstudy.mpesa
 
 import com.androidstudy.daraja.constants.TransactionType
 
-
 object Config {
     var CONSUMER_KEY = "Uku3wUhDw9z0Otdk2hUAbGZck8ZGILyh"
```

Comment:
Can we have this on `local.properties` file or is it essential they be part of the code?

Reply:
We can move it actually, that would be the best place. But then update the README to reflect that, so that a new developer would understand where to get the keys